CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
This `salary` field appears to be a temporary or placeholder variable and violates naming conventions. It should be removed before merging to maintain code cleanliness and avoid confusion.

```suggestion
// TODO: Remove before merging
```